templates/base.html.twig

```diff
@@ -46,6 +46,8 @@
         {% endblock %}
     </footer>
 
+    {{ render_hinclude('/admin/website-toolbar') }}
```

Comment:
Its very important to use here `hinclude` or any other `ajax` request to render the html. You can  not use here `include` or the normal `render` function as the `Website` does know nothing about the `Admin` and so you can not do any security checks for the admin users here.